docs/release_notes.md

```diff
@@ -1,5 +1,67 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Added
+
+#### Core
+
+- Workflow execution on a dedicated thread. This allows users to inspect and author different workflows while executing.
+- `SHIFT_SINGLE_THREADED` environment variable to switch the threaded execution behaviour.
+- Workflow execution progress bar in the Shift UI.
+- Board styling for higlighting the execution status of the workflow.
+- Ability to request the interruption of an executing workflow with the "Clear Execution" button. The button will now clear the workflow only if pressed while no execution is running on the current board.
+- Ability to rename a node by double clicking on its name with the left mouse button.
+
+#### Integration & Resources
+
+- Nuke: Menu file to setup Shift tools inside Nuke.
+- Nuke (Beta): "ShiftWorkflow" Nuke node to execute Shift workflow inside the Nuke node graph.
+
+#### API
+
+- Added `shift.utils.dcc_threading` module containing utilities to handle threaded execution in DCC context
+- Added `shift.utils.decorators` module with `mainThread` decorator for SOperator.execute methods to force execution on main thred in Maya and Nuke
```

Comment:
Remember to add the full stop at the end of the sentence to be aligned with the rest.
```suggestion
- Added `shift.utils.dcc_threading` module containing utilities to handle threaded execution in DCC context.
- Added `shift.utils.decorators` module with `mainThread` decorator for SOperator.execute methods to force execution on main thred in Maya and Nuke.
```